src/breadcrumb/breadcrumb.scss

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+
+@mixin iui-breadcrumb {
+  @include iui-reset;
+  @include iui-font-family;
+  display: flex;
+  align-items: center;
+
+  > .iui-button {
+    margin-right: $iui-s;
+  }
+  
+  ol {
+    @include iui-reset;
+    list-style-type: none;
+    display: flex;
+    align-items: center;
+    user-select: none;
+
+    li {
+      overflow: hidden;
+      white-space: nowrap;
+      text-overflow: ellipsis;
+      max-width: $iui-3xl * 2;
+
+      a:not([aria-current]) {
+        @include iui-anchor;
+      }
+
+      [aria-current] {
+        text-decoration: none;
+        @include iui-focus;
+        @include themed {
+          color: t(iui-text-color);
+        }
+      }
+
+      &:not(:first-child)::before {
+        content: '\203A';
+        font-size: $iui-font-size-subheading;
+        margin: 0 $iui-m;
+        @include themed {
+          color: t(iui-text-color-muted);
+        }
+      }
+    }
+  }
+
+  .iui-input-container {
```

Comment:
🤔

Reply:
Should I remove the margin from the `.iui-input-container` SCSS?  I could add the 11px bottom margin within the HTML page's `<style>` so tests don't fail.

Reply:
Will make this change in #97 .